.ruby-version

```diff
@@ -1 +1 @@
-3.0
+3.0.0
```

Comment:
```suggestion
3.0.4
```

because that's what's used in [CircleCI](https://github.com/18F/identity-oidc-sinatra/blob/main/.circleci/config.yml#L10) and in the [main idp repo](https://github.com/18F/identity-idp/blob/main/.ruby-version#L1)

Reply:
I like this idea!  I'll make the change and push it up to github.